src/components/Facility/ConsultationDetails/PrintQuestionnaireQuestionnaireResponses.tsx

```
@@ -0,0 +1,381 @@
+    !Array.isArray(value) &&
+    !(value instanceof Date)
+  ) {
+    return JSON.stringify(value);
+  }
+
+  switch (type) {
+    case "dateTime":
+      return formatDateTime(value as string);
+    case "choice":
+      return properCase(value.toString());
+    case "decimal":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)